.github/workflows/database-health-check.yml

```
@@ -0,0 +1,168 @@
+        node-version: '18'
+        cache: 'bun'
+        
+    - name: Install Bun
+      uses: oven-sh/setup-bun@v1
+      
+    - name: Install dependencies
+      run: bun install --frozen-lockfile
+      
+    - name: Run database health check
+      run: bunx tsx -e "
+      import { createClient } from '@supabase/supabase-js';
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 32: could not find expected ':'

(syntax-check)